tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/ServerPreviewTest.java

```diff
@@ -0,0 +1,56 @@
+package org.jboss.hal.testsuite.test.runtime;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.meta.token.NameTokens;
+import org.jboss.hal.resources.Ids;
+import org.jboss.hal.testsuite.Console;
+import org.jboss.hal.testsuite.Random;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.fragment.AddResourceDialogFragment;
+import org.jboss.hal.testsuite.fragment.finder.FinderFragment;
+import org.jboss.hal.testsuite.util.ServerEnvironmentUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+
+@RunWith(Arquillian.class)
+public class ServerPreviewTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final ServerEnvironmentUtils serverEnvironmentUtils = new ServerEnvironmentUtils(client);
+
+    @AfterClass
+    public static void tearDown() throws IOException {
+        client.close();
+    }
+
+    @Drone
+    private WebDriver browser;
+
+    @Inject
+    private Console console;
+
+    @Test
+    public void editUrl() throws IOException {
+        String url = Random.name();
+        FinderFragment fragment = console.finder(NameTokens.RUNTIME);
+        fragment.column(Ids.STANDALONE_SERVER_COLUMN)
```

Comment:
This navigation is standalone specific so I suggest to either add a new `Standalone` category and annotate this test case with it or change it to be able to run for domain mode as well.